contracts/staking/TokenStaking.sol

```diff
@@ -269,7 +277,7 @@ contract TokenStaking is Ownable, IStaking, Checkpoints {
     ///         stake amount from KEEP staking contract. Can be called by
     ///         anyone.
     function stakeKeep(address operator) external override {
-        require(operator != address(0), "Operator must be specified");
+        require(operator != address(0), "Parameters must be specified");
```

Comment:
Is it more efficient for the contract size to have the same string everywhere? 🤔

Reply:
Yep, now it's stored only once, it's the most efficient optimization here 😄